modules/twinklespeedy.js

```diff
@@ -232,6 +235,21 @@ Twinkle.speedy.initDialog = function twinklespeedyInitDialog(callbackfunc) {
 			}
 		]
 	});
+	tagOptions.append({
+		type: 'checkbox',
+		list: [
+			{
+				label: 'Tag for creation protection (salting) as well',
```

Comment:
I know I suggested this language, but it still feels awkward.  I don't have a suggestion — I think it's good to have something like "also" or "as well" but it's good to have "Tag" be the first word — but just wanted to note it.

Reply:
Maybe "Request creation protection (salting) as well"?

Reply:
I think having the "tag" in there is helpful, so it's clear we're not making an RfPP request.  Maybe that's over the top, though?  It's probably fine as is, just making life tricky.